scopes/ui-foundation/uis/side-bar/component-tree/component-tree.tsx

```diff
@@ -35,7 +38,11 @@ export function ComponentTree({
     <div style={indentStyle(1)}>
       <TreeNodeContext.Provider value={TreeNode}>
         <TreeContextProvider onSelect={onSelect} selected={selected}>
-          <RootNode node={rootNode} depth={1} />
+          {rootNode.children?.length === 0 ? (
```

Comment:
this is not the place to put this. look at `workspace-components.drawer.tsx` for the workspace and `components.drawer.tsx` for scope.

Reply:
if we do it like you suggested, we cant show different messages for scope/workspace if we want and also it will not be reusable anywhere else

Reply:
done